package.json

```diff
@@ -43,7 +43,7 @@
     "@navikt/familie-http": "6.0.0",
     "@navikt/familie-ikoner": "7.0.0",
     "@navikt/familie-logging": "6.0.0",
-    "@navikt/familie-skjema": "7.0.8",
+    "@navikt/familie-skjema": "7.0.3",
```

Comment:
Det er _noe_ som har skjedd fra v7.0.3 til v7.0.8 som gjør at vi får "request is not a function" på "neste steg" på simuleringssiden. Siden jeg egentlig ikke trenger å oppgradere familie-skjema i denne PRen, men kun gjorde det fordi jeg så vi ikke var up-to-date, reverterer jeg denne oppgraderingen enn så lenge